stable/concourse/README.md

```diff
@@ -244,3 +270,26 @@ web:
         hosts:
           - concourse.domain.com
 ```
+
+
+### PostgreSQL
+
+By default, this chart will use a PostgreSQL database deployed as a chart dependency. You can also bring your own PostgreSQL. To do so, set the following in your custom `values.yaml` file:
+
+```yaml
+## Configuration values for the postgresql dependency.
+## ref: https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md
+##
+postgresql:
+
+  ## Use the PostgreSQL chart dependency.
+  ## Set to false if bringing your own PostgreSQL.
+  ##
+  enabled: false
+
+  ## If bringing your own PostgreSQL, the full uri to use
+  ## e.g. postgres://concourse:changeme@my-postgres.com:5432/concourse?sslmode=require
+  ##
+  uri: postgres://concourse:changeme@my-postgres.com:5432/concourse?sslmode=require
```

Comment:
Would it be possible to specify a uri with credentials that I do not know upfront? E.g. when existing postgres pulls password from a `secret` object in cluster